mobile-api/package.json

```
@@ -9,18 +9,23 @@
     "npm": ">=8"
   },
   "scripts": {
-    "dev": "nodemon src/index.ts"
+    "build": "tsc",
+    "dev": "nodemon src/index.ts",
+    "start": "node src/index.js"
```

```suggestion
    "start": "NODE_ENV=production node prod/index.js"
```
Since the compiled code lives there now.

---

In the final container the transpiled files are copied to the `src` folder - https://github.com/freeCodeCamp/mobile/pull/194/files#diff-a4a59dd5c7c642b12a892c609058d66dfbfe9d1a537b39ad272690531b9bef9aR15 .

---

The issue with that is that `npm run start` then fails when run locally.  How about just using the `prod` folder inside the container?